src/neptune/management/internal/utils.py

```diff
@@ -50,3 +51,8 @@ def normalize_project_name(name: str, workspace: Optional[str] = None):
     extracted_workspace_name, extracted_project_name = extract_project_and_workspace(name=name, workspace=workspace)
 
     return f"{extracted_workspace_name}/{extracted_project_name}"
+
+
+class WorkspaceMemberRole(Enum):
+    MEMBER = "member"
+    ADMIN = "admin"
```

Comment:
That is a project role.

Reply:
I know it's for workspaces and from the user perspective/UI it should be `Admin` but we're passing this value in this PR directly to the our backend API. On the API level I think it's `owner` instead of `admin` and it should be mapped in-between this enum and calling an API endpoint.

Reply:
Are you sure about that? I was able to call API with `Admin` value and then confirmed that the correct invitation was created

Reply:
Yes, I'm quite sure. UI is making a request with `owner` and I see that backend only allows `member`/`owner`.
![Screenshot 2023-05-08 at 14 23 58](https://user-images.githubusercontent.com/917619/236823051-7bc9f826-bf9b-490e-a52f-1facdc045dc4.png)